src/primitive_value.rs

```diff
@@ -71,6 +74,58 @@ pub enum PrimitiveValue {
     BigDecimal(Option<Box<BigDecimal>>),
 }
 
+impl QueryValue for PrimitiveValue {
+    fn query_value(&self, query_builder: &dyn QueryBuilder) -> String {
+        match self {
+            PrimitiveValue::Bool(value) => value.query_value(query_builder),
+            PrimitiveValue::TinyInt(value) => value.query_value(query_builder),
+            PrimitiveValue::SmallInt(value) => value.query_value(query_builder),
+            PrimitiveValue::Int(value) => value.query_value(query_builder),
+            PrimitiveValue::BigInt(value) => value.query_value(query_builder),
+            PrimitiveValue::TinyUnsigned(value) => value.query_value(query_builder),
+            PrimitiveValue::SmallUnsigned(value) => value.query_value(query_builder),
+            PrimitiveValue::Unsigned(value) => value.query_value(query_builder),
+            PrimitiveValue::BigUnsigned(value) => value.query_value(query_builder),
+            PrimitiveValue::Float(value) => value.query_value(query_builder),
+            PrimitiveValue::Double(value) => value.query_value(query_builder),
+            PrimitiveValue::String(value) => value.as_deref().cloned().query_value(query_builder),
```

Comment:
I couldn't find a way to do this without `.cloned()`. I think some changes need to be made in `impl<T> QueryValue for Option<T>`, but I had issues getting it to work.